nuxeo-core/nuxeo-core-bulk/src/test/java/org/nuxeo/ecm/core/bulk/action/TestWordCountAction.java

```diff
@@ -18,16 +18,16 @@
  */
 package org.nuxeo.ecm.core.bulk.action;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-import static org.nuxeo.ecm.core.bulk.message.BulkStatus.State.COMPLETED;
-
```

Comment:
strange move

Reply:
yes bad formatter config fixed in next push